HLTrigger/Configuration/test/test_hltPhase2BaselineTracking.py

```diff
@@ -0,0 +1,77 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("HLTX")
+
+### Load all ESSources, ESProducers and PSets
+process.load("HLTrigger.Configuration.Phase2.hltPhase2Setup_cff")
+
+### ES Prefers... would "prefer" to have them somewhere else
+process.prefer("es_hardcode")
+process.prefer("ppsDBESSource")
+process.prefer("siPixelFakeGainOfflineESSource")
+
+### Don't rerun the module that makes TrackTrigger tracks
+# process.TTTracksFromTrackletEmulation = cms.EDProducer
+
+process.offlineBeamSpot = cms.EDProducer("BeamSpotProducer")
+
+process.hltPhase2GeneralTracks = cms.EDProducer(
+    "TrackListMerger",
+    Epsilon=cms.double(-0.001),
+    FoundHitBonus=cms.double(5.0),
+    LostHitPenalty=cms.double(5.0),
+    MaxNormalizedChisq=cms.double(1000.0),
+    MinFound=cms.int32(3),
+    MinPT=cms.double(0.9),
+    ShareFrac=cms.double(0.19),
+    TrackProducers=cms.VInputTag(
+        "hltPhase2L1CtfTracks",
+        "hltPhase2InitialStepTrackSelectionHighPurity",
+        "hltPhase2HighPtTripletStepTrackSelectionHighPurity",
+    ),
+    allowFirstHitShare=cms.bool(True),
+    copyExtras=cms.untracked.bool(True),
+    copyMVA=cms.bool(False),
+    hasSelector=cms.vint32(0, 0, 0),
+    indivShareFrac=cms.vdouble(1.0, 1.0, 1.0),
+    makeReKeyedSeeds=cms.untracked.bool(False),
+    newQuality=cms.string("confirmed"),
+    selectedTrackQuals=cms.VInputTag(
+        cms.InputTag(""),
+        cms.InputTag("hltPhase2InitialStepTrackSelectionHighPurity"),
+        cms.InputTag("hltPhase2HighPtTripletStepTrackSelectionHighPurity"),
+    ),
+    setsToMerge=cms.VPSet(cms.PSet(pQual=cms.bool(True), tLists=cms.vint32(0, 1))),
+    trackAlgoPriorityOrder=cms.string("trackAlgoPriorityOrder"),
+    writeOnlyTrkQuals=cms.bool(False),
+)
+
+process.load("HLTrigger.Configuration.Phase2.trackerLocalReco_cff")
```

Comment:
I sort of expected that only hlt-prefixed ED modules will be used.
What is the reason to keep this setup?

Reply:
(I don't find a way to link to the answer to Martin below, so I repeat it here)

I was worried about the offline local reco differing from the online, but for trackerLocalReco I reckon the chances of this happening are very low. I will rename it hltPhase2TrackerLocalReco, import from the offline and modify the parameters if needed.

Reply:
This is addressed now!